### DISCUSSION TITLE
Is there an easy wrapper for wgpu?

### DISCUSSION THREAD

**Discussion by Raj2032** [gfx-rs/wgpu#2705] [Q&A] [answered]
Hey guys, I was wondering if there was like some easy wrapper for wgpu which draws shapes and stuff, just like [raylib](https://www.raylib.com/) which is a wrapper for opengl?

**Comment by Cactice** [comment, marked as answer]
I guess you got answers on reddit.
https://www.reddit.com/r/rust/comments/v10iwp/is_there_an_easy_wrapper_for_wgpu/

**Comment by Raj2032** [comment]
Thanks mate
